app/editor/src/hooks/useAdmin.ts

```diff
@@ -0,0 +1,10 @@
+import { Claim, useKeycloakWrapper } from 'tno-core';
+
+/**
+ * Hook to determine if a user is an admin based off their claims
+ * @returns boolean value indicating if the user is an admin
+ */
+export const useAdmin = () => {
```

Comment:
This hook is a bit overkill.  Unless it has more functionality it is more intuitive for us to just use the `useKeycloakWrapper` hook.

Reply:
Sounds good, think I've done this overkill one twice now 😛 getting deja vu